docs/content/latest/explore/multi-region-deployments/read-replicas.md

```diff
@@ -0,0 +1,281 @@
+```
+
+Output:
+
+```
+0 [main] INFO com.yugabyte.sample.Main - Starting sample app...
+22 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Using NO UUID
+27 [main] INFO com.yugabyte.sample.common.CmdLineOpts - App: CassandraKeyValue
+27 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Run time (seconds): -1
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Adding node: 127.0.0.1:9042
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Adding node: 127.0.0.2:9042
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Adding node: 127.0.0.3:9042
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Num reader threads: 1, num writer threads: 1
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Num unique keys to insert: 2
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Num keys to update: 1999998
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Num keys to read: 1500000
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Value size: 1024
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Restrict values to ASCII strings: false
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Perform sanity check at end of app run: false
+28 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Table TTL (secs): -1
+29 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Local reads: true
+29 [main] INFO com.yugabyte.sample.common.CmdLineOpts - Read only load: false
+30 [main] INFO com.yugabyte.sample.apps.AppBase - Creating Cassandra tables...
+67 [main] INFO com.yugabyte.sample.apps.AppBase - Connecting with 4 clients to nodes: /127.0.0.1:9042,/127.0.0.2:9042,/127.0.0.3:9042
+751 [main] INFO com.yugabyte.sample.apps.AppBase - Created a Cassandra table using query: [CREATE TABLE IF NOT EXISTS CassandraKeyValue (k varchar, v blob, primary key (k));]
+5773 [Thread-1] INFO com.yugabyte.sample.common.metrics.MetricsTracker - Read: 4029.25 ops/sec (0.24 ms/op), 20221 total ops | Write: 1486.29 ops/sec (0.67 ms/op), 7440 total ops | Uptime: 5021 ms | maxWrittenKey: 1 | maxGeneratedKey: 2 | 
+10778 [Thread-1] INFO com.yugabyte.sample.common.metrics.MetricsTracker - Read: 4801.54 ops/sec (0.21 ms/op), 44256 total ops | Write: 1637.30 ops/sec (0.61 ms/op), 15635 total ops | Uptime: 10026 ms | maxWrittenKey: 1 | maxGeneratedKey: 2 | 
+```
+
+The following illustration demonstrates the reads spread across all the replicas for the tablet visible via [Yugabyte Platform](https://docs.yugabyte.com/latest/yugabyte-platform/):
+
+![img](/images/explore/multi-region-deployments/read-replicas3.png)
+
+## Using Read Replicas
+
+The following commands add three new nodes, and then add these nodes to a read replica cluster in region `r2`:
+
+```shell
+./bin/yb-ctl add_node --placement_info "c.r2.z21" --tserver_flags "placement_uuid=rr"                                      ./bin/yb-ctl add_node --placement_info "c.r2.z22" --tserver_flags "placement_uuid=rr"
```

Comment:
Missing newline?

Reply:
Yes, thanks for catching it.
Also, `add_read_replica_placement_info c.r2.z21:1,c.r2.z22:1,c.r2.z23:1 3 rr` is an output, right? So I moved it out of the command block.